test/installer/scripts/prep_omsadmin.sh

```diff
@@ -20,6 +20,19 @@ sed -i s,INSTALL_INFO=.*,INSTALL_INFO=$BASE_DIR/installer/conf/installinfo.txt,1
 sed -i s,AGENT_USER=.*,AGENT_USER=`id -un`,1 $OMSADMIN
 sed -i s,AGENT_GROUP=.*,AGENT_GROUP=`id -gn`,1 $OMSADMIN
 sed -i s,METACONFIG_PY=.*,METACONFIG_PY=$TESTDIR/not_a_file,1 $OMSADMIN
+sed -i s,PROCESS_STATS=.*,PROCESS_STATS=$TESTDIR/process_stats,1 $OMSADMIN
+
+# Don't run omi in a test since it may not be installed.
+# The output generated by omi is mocked in process_stats.
+sed -i s,/opt/omi/bin/omicli.*,,1 $OMSADMIN
+
+cat <<EOF > $TESTDIR/process_stats
+    PercentUserTime=2
+    PercentPrivilegedTime=1
+    UsedMemory=27712
+    PercentUsedMemory=3
+EOF
+
```

Comment:
I didn't realize that a SELECT was being done, I thought it was just an enumeration (which would result in very different output, but output that the grep should still work with).

Sorry for the confusion!